pandas/tests/arithmetic/conftest.py

```diff
@@ -9,6 +9,18 @@
     UInt64Index,
 )
 import pandas._testing as tm
+from pandas.core.computation import expressions as expr
+
+
+@pytest.fixture(
+    autouse=True, scope="module", params=[0, 1000000], ids=["numexpr", "python"]
+)
+def switch_numexpr_min_elements(request):
+    _MIN_ELEMENTS = expr._MIN_ELEMENTS
+    expr._MIN_ELEMENTS = request.param
+    yield request.param
+    expr._MIN_ELEMENTS = _MIN_ELEMENTS
```

Comment:
can we get rid of some of the setup/teardown in test_expressions with this?

Reply:
Potentially something similar could be used there as well, yes. But this PR is focusing on the `tests/arithmetic/` tests, there is https://github.com/pandas-dev/pandas/issues/40497 as general issue to modernize `test_expressions.py`